resources/views/fr/Emails/Auth/Reminder.blade.php

```diff
@@ -0,0 +1,11 @@
+@extends('en.Emails.Layouts.Master')
+
+@section('message_content')
+    <div>
+        Hello,<br><br>
```

Comment:
?

Reply:
What do you mean by "?" ?

Reply:
I would prefer more uniformity over formality but I am not able to decide.